IQKeyBoardManager/IQToolbar/IQUIView+IQKeyboardToolbar.m

```diff
@@ -148,7 +148,16 @@ - (void)addRightButtonOnKeyboardWithText:(NSString*)text target:(id)target actio
     [items addObject:nilButton];
     
     //  Create a done button to show on keyboard to resign it. Adding a selector to resign it.
-    IQBarButtonItem *doneButton =[[IQBarButtonItem alloc] initWithTitle:text style:UIBarButtonItemStyleDone target:target action:action];
+    IQBarButtonItem *doneButton;
+    UIImage *doneImage = [IQKeyboardManager sharedManager].doneImage;
```

Comment:
Thank you for the feedback @hackiftekhar. I'll have a look at it when I have the time and try those changes.